src/main/resources/application.properties

```diff
@@ -1 +1,8 @@
 spring.application.name=online-book-store
+spring.datasource.url=jdbc:mysql://localhost:3306/book_store_db
+spring.datasource.username=root
+spring.datasource.password=123sqlBod321
```

Comment:
do not push credentials, it's a good practice even if it is your local DB on your machine